lfs.c

```diff
@@ -5449,6 +5512,10 @@ static int lfs1_unmount(lfs_t *lfs) {
 /// v1 migration ///
 static int lfs_rawmigrate(lfs_t *lfs, const struct lfs_config *cfg) {
     struct lfs1 lfs1;
+    if(cfg->block_count == 0){
```

Comment:
changing it. I also wanted to confirm with you if this assertion is correct? I didn't quite grok all of what the superblock state is before/after migration. So this assertion was a bit of laziness on my part ;P

Reply:
Hmmm, I suppose, in theory, you could find the `block_count` from the v1 superblock...

It's really not worth adding this feature though. I believe the v1 code is basically legacy at this point, and any v1 filesystems that can be migrated hopefully have been migrated.

At least, any applications written to expect an unknown block_count should probably be written for v2 since this was never a feature for v1.